app/src/main/java/org/wikipedia/search/SearchResultsFragment.java

```diff
@@ -66,6 +70,7 @@
     }
 
     private static final int BATCH_SIZE = 20;
+    private static final int LARGE_BATCH_SIZE = 100;
```

Comment:
Why do we need such a large batch size here? This is pretty resource-intensive for the server.

Reply:
Make sense. Removed.